CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.120220906]
+
+- Sample release

```

Comment:
💡 Nit: Might be something like - `Updating the code samples for CLI.` thanks.

Reply:
Updated, Thank you @Tatsinnit

Reply:
@Tatsinnit You want to separate out samples to separate project? it was actually in separate mindaro project we brought it to bridge to keep them all together. Maybe we can create it as bridge-to-Kubernetes-samples?

Reply:
💡☕️ Yeah, just idea, let's merge this for now and we can chat about it more clearly on side, if the discussion will hold off this improvement. Thank you for this.